packages/cli/src/cmds/validator/import.ts

```diff
@@ -26,7 +26,7 @@ Ethereum Foundation utility.",
 
   examples: [
     {
-      command: "validator import --network hoodi --importKeystores $HOME/staking-deposit-cli/validator_keys",
+      command: "validator import --network hoodi --importKeystores $HOME/staking-deposit-cli/validator_keys --keystoresLockFilePath $HOME/staking-deposit-cli/validator_keys.lock",
```

Comment:
this shouldn't be updated, we wanna keep the example simple, but noting here that `keystoresLockFilePath` should be a directory, not a file path